frontend/packages/core/src/button.tsx

```diff
@@ -18,18 +18,18 @@ const COLORS = {
   primary: {
     background: {
       primary: "#3548D4",
-      hover: "#7680E8",
-      active: "#1B31C6",
+      hover: "#2D3DB4",
+      active: "#2938A5",
       disabled: "#7680E8",
```

Comment:
primary disabled is now
```suggestion
      disabled: "#E7E7EA",
```